lib/chef/provider/chef_group.rb

```diff
@@ -15,13 +15,13 @@ def whyrun_supported?
       if differences.size > 0
         description = [ "update group #{new_resource.name} at #{rest.url}" ] + differences
         converge_by description do
-          rest.put("groups/#{new_resource.name}", normalize_for_put(new_json))
+          rest.put("organizations/#{new_resource.org}/groups/#{new_resource.name}", normalize_for_put(new_json))
```

Comment:
Things in Cheffish don't generally follow this pattern (though it's not unreasonable).  Generally, they expect chef_server_url to be https://<chef server>/organizations/org .  That's a general Chef pattern.